spec/src/main/asciidoc/fallback.asciidoc

```diff
@@ -94,5 +94,28 @@ When `fallbackMethod` is used a `FaultToleranceDefinitionException` will be thro
 The parameter `value` and `fallbackMethod` on `@Fallback` cannot be specified at the same time.
 Otherwise, the `FaultToleranceDefinitionException` exception will be thrown.
 
-The fallback should be triggered when an exception occurs.
-For instance, `BulkheadException`, `CircuitBreakerOpenException`, `TimeoutException` should trigger the fallback.
+The fallback might be triggered when an exception occurs (e.g. `BulkheadException`, `CircuitBreakerOpenException`, `TimeoutException`, etc), detailed below.
+When a method returns and the Fallback policy is present, the following rules are applied:
```

Comment:
Added a new section

Reply:
There are 2 sections under `=== Fallback usage`, called `==== Specify a FallbackHandler class` and `==== Specify the fallbackMethod`. If we make this one `==== Specify the criteria for triggering Fallback`, it looks like it's a 3rd method of specifying fallback.

I'd call it `=== Criteria for triggering fallback`, so that:

1. it's on the same level as `=== Fallback usage`;
2. the name doesn't look like it's something the user needs to do.

Reply:
I agree with @Azquelt ! I think it looks better as it stands and still fall into the usage category as these are new parameters on the annotation of fallback.